.gitignore

```diff
@@ -10,3 +10,4 @@
 /dist/gh-pages/
 /tmp-test-bundle.js
 /coverage/
+.idea/*
```

Comment:
I'd rather not start committing this for everyone's choice of editor.

Could you remove this entry?